src/Window.cpp

```diff
@@ -589,10 +581,9 @@ void CWindow::updateDynamicRules() {
 // it is assumed that the point is within the real window box (m_vRealPosition, m_vRealSize)
 // otherwise behaviour is undefined
 bool CWindow::isInCurvedCorner(double x, double y) {
-    static auto* const ROUNDING   = &g_pConfigManager->getConfigValuePtr("decoration:rounding")->intValue;
-    static auto* const BORDERSIZE = &g_pConfigManager->getConfigValuePtr("general:border_size")->intValue;
+    static auto* const ROUNDING = &g_pConfigManager->getConfigValuePtr("decoration:rounding")->intValue;
 
-    if (BORDERSIZE >= ROUNDING || ROUNDING == 0)
+    if (getRealBorderSize() >= *ROUNDING || *ROUNDING == 0)
```

Comment:
not sure if this correct, but doesn't make sense otherwise

Reply:
we could switch this over to `CWindow::rounding()` while we're at it. But yeah, looks alright.